libr/include/r_bp.h

```diff
@@ -84,9 +84,9 @@ typedef struct r_bp_t {
 } RBreakpoint;
 
 enum {
-	R_BP_PROT_READ = 1,
+	R_BP_PROT_EXEC = 1,
```

Comment:
Osx swaps this. So it shouldnt hurt this change if its handled properly.. lets mwrge and see whats going on